cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -0,0 +1,277 @@
+                        _.map(_.range(expectedXBlocksCount), function() {
+                            return 'Press button to see ' + category + ' childs';
+                        })
+                    );
+                    expect(displayedInfo.forwardButtonCount).toEqual(expectedXBlocksCount);
+                }
+            };
+
+            verifyBreadcrumbViewInfo = function(category, xblockIndex) {
+                var displayedBreadcrumbs = moveXBlockBreadcrumbView.$el.find('.breadcrumbs .bc-container').map(
+                    function() { return $(this).text().trim(); }
+                ).get(),
+                    categories = _.keys(parentToChildMap).concat(['component']),
+                    visitedCategories = categories.slice(0, _.indexOf(categories, category));
+
+                expect(displayedBreadcrumbs).toEqual(
+                    _.map(visitedCategories, function(cat) {
+                        return cat === 'course' ?
+                            'Course Outline' : cat + '_display_name_' + xblockIndex;
+                    })
+                );
+            };
+
+            clickForwardButton = function(buttonIndex) {
+                moveXBlockListView.$el.find('[data-item-index="' + buttonIndex + '"] button').click();
+            };
+
+            clickBreadcrumbButton = function() {
+                moveXBlockBreadcrumbView.$el.find('.bc-container button').last().click();
+            };
+
+            nextCategory = function(direction, category) {
+                return direction === 'forward' ? parentToChildMap[category] : _.invert(parentToChildMap)[category];
+            };
+
+            verifyXBlockInfo = function(options, category, buttonIndex, direction, hasCurrentLocation) {
+                var expectedXBlocksCount = options[category];
+
+                verifyListViewInfo(category, expectedXBlocksCount, hasCurrentLocation);
+                verifyBreadcrumbViewInfo(category, buttonIndex);
+
+                if (direction === 'forward') {
+                    if (category === 'component') {
+                        return;
+                    }
+                    clickForwardButton(buttonIndex);
+                } else if (direction === 'backward') {
+                    if (category === 'section') {
+                        return;
+                    }
+                    clickBreadcrumbButton();
+                }
+                category = nextCategory(direction, category);  // eslint-disable-line no-param-reassign
+
+                verifyXBlockInfo(options, category, buttonIndex, direction, hasCurrentLocation);
+            };
+
+            it('renders views with correct information', function() {
+                var outlineOptions = {section: 1, subsection: 1, unit: 1, component: 1},
+                    outline = createCourseOutline(outlineOptions);
+
+                renderViews(outline);
+                verifyXBlockInfo(outlineOptions, 'section', 0, 'forward', false);
```

Comment:
`hasCurrentLocation` is set to false to avoid checking the current location text as we have a separate test for this.